content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,100 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers from users to controllers.
+
+First, the user delegates ownership of the field to a temporary, uniquely named,
+field manager. Next, the user signals to the controller that it should take
+ownership of the field, usually by setting some other field (perhaps in some
+other object) that the controller observes. Last, controller then claims ownership
+of the field by updating the object or applying a configuration containing the
+field.
+
+Transferring fields to controllers this way ensures the field is not accidently
+removed from the live object, or reset to a default value, at any point of time
+during the transfer.
+
+This approach can be used, for example, to safely transfer ownership of the
+`replicas` field from a user to a controller while enabling Horizontal Pod
+Autoscaling on an object.
+
+Say a user has defined deployment with `replicas` set to 100:
+
+{{< codenew file="application/ssa/nginx-deployment.yaml" >}}
+
+And the user created the deployment using server side apply like so:
+
+```shell
+kubectl apply -f application/ssa/nginx-deployment.yaml --server-side
+```
```

Comment:
Turn on HPA at this point (move that step here). It will motivate the next thing.